packages/alfa-rules/src/sia-r28/rule.ts

```diff
@@ -1,19 +1,16 @@
 import { Rule, Diagnostic } from "@siteimprove/alfa-act";
-import { Name } from "@siteimprove/alfa-aria";
 import { Element, Namespace } from "@siteimprove/alfa-dom";
-import { Iterable } from "@siteimprove/alfa-iterable";
 import { Predicate } from "@siteimprove/alfa-predicate";
 import { Err, Ok } from "@siteimprove/alfa-result";
 import { Page } from "@siteimprove/alfa-web";
 
 import { expectation } from "../common/expectation";
 
-import { hasAccessibleName } from "../common/predicate/has-accessible-name";
+import { hasNonEmptyAccessibleName } from "../common/predicate/has-non-empty-accessible-name";
 import { hasInputType } from "../common/predicate/has-input-type";
 import { isIgnored } from "../common/predicate/is-ignored";
```

Comment:
Unrelated to this PR, but this makes me wonder whether we should have a `common/predicate.ts` to re-export everything from `common/predicate/*`, probably in a namespace, and `const { predicate1, predicate2, predicate3 } = CommonPredicate` or similar to group these imports
🤔